spark-template-jade/src/main/java/spark/template/jade/JadeTemplateEngine.java

```diff
@@ -50,6 +50,24 @@ public JadeTemplateEngine(String templateRoot) {
     }
 
     /**
+     * Construct a jade template engine from a raw JadeConfiguration
+     *
+     * @param configuration the raw JadeConfiguration to use
+     */
+    public JadeTemplateEngine(JadeConfiguration configuration) {
+        this.configuration = configuration;
+    }
+
+    /**
+     * Access to the internal JadeConfiguration
+     *
+     * @return The JadeConfiguration used by this engine
+     */
+    public JadeConfiguration getConfiguration() {
```

Comment:
To follow spark public API naming conventions it should be renamed to `configuration()`.
Otherwise :+1: